shell/pom.xml

```diff
@@ -69,5 +69,45 @@
       <artifactId>alluxio-core-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
currently, the UFS modules are not included into and thus not available at runtime in the assembly-client jar, which is the Java binary for running shell commands. One option is to add these UFS dependencies into the assembly-client pom file `assembly/client/pom.xml`, however, I am not convinced this is the right approach as this put UFS classes into client, defecting our purpose for delegation